modules/products/confluence/main.tf

```diff
@@ -28,3 +28,18 @@ module "database" {
   db_master_username      = var.db_master_username
   db_master_password      = var.db_master_password
 }
+
+module "nfs" {
+  source = "../../AWS/nfs"
+
+  namespace               = var.namespace
+  product                 = local.product_name
+  requests_cpu            = var.nfs_requests_cpu
+  requests_memory         = var.nfs_requests_memory
+  limits_cpu              = var.nfs_limits_cpu
+  limits_memory           = var.nfs_limits_memory
+  capacity                = var.shared_home_size
```

Comment:
Sorry I couldn't check the previous PR but do you know why we have two variables for shared_home_size?

Reply:
So basically `capacity` is used when creating PVC for NFS server pod, and `shared_home_size` when creating PVC for Jira & Confluence pod.

Reply:
In this case is it better to rename `capacity` to `nfs_storage_capacity`?